wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/MessageActions/CompleteReactionPicker/ReactionSectionViewController.swift

```diff
@@ -55,10 +56,8 @@ final class ReactionSectionViewController: UIViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        guard let firstButton = typesByButton.first?.key else { return }
-        for button in sectionButtons {
-            button.isSelected = (button == firstButton)
-        }
+        guard let firstType = types.first else { return }
+        selectedType = firstType
```

Comment:
Why don't you want to replace selectedType in the `init` method? You set a value for selectedType in init too.

Reply:
setting `selectedType` updates all buttons visual properties and it's not working until view is completely loaded

Reply:
So, can you remove it from `init` since it will be overrided every time in `viewWillAppear`?